redbox/model_db.py

```diff
@@ -5,23 +5,21 @@
 
 from sentence_transformers import SentenceTransformer
 
-from redbox.models import ModelInfo, Settings
+from redbox.models import ModelInfo
 from redbox.models.llm import Embedding, EmbeddingResponse
 
 MODEL_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "models")
 
 log = logging.getLogger()
 log.setLevel(logging.INFO)
 
-env = Settings()
-
 
 class SentenceTransformerDB(collections.UserDict):
```

Comment:
i have removed the dependancy on the Settings here so that it can be called without an `.env` existing, e.g. at build time